src/test/ensemble_flow_custom_node_tests.cpp

```diff
@@ -4137,9 +4137,9 @@ TEST_F(EnsembleFlowCustomNodeAndDynamicDemultiplexerLoadConfigThenExecuteTest, J
     ASSERT_NE(inputs.find(pipelineInputName), inputs.end());
     ASSERT_NE(outputs.find(pipelineOutputName), outputs.end());
     auto& input_A = inputs.at(pipelineInputName);
-    EXPECT_EQ(input_A->getEffectiveShape(), shape_t({1, 10}));
+    EXPECT_EQ(input_A->getShape_3(), Shape({1, 10}));
     auto& output = outputs.at(pipelineOutputName);
-    EXPECT_EQ(output->getEffectiveShape(), shape_t({0, 1, 10}));
+    EXPECT_EQ(output->getShape_3(), Shape({-1, 1, 10}));
```

Comment:
The whole point of having hidden constant `DYNAMIC_DIMENSION` is to be able to change it to any value at any time. This test will fail if we change it. Use:
```
Shape{Dimension::any(), 1, 10}
```